test/infer/test_autoguide.py

```diff
@@ -837,7 +837,11 @@ def model():
             batch = numpyro.subsample(data, event_dim=0)
             numpyro.sample("x", dist.Normal(batch, 1))
 
-    guide = AutoSemiDAIS(model, model, None)
+    def create_plates():
+        return numpyro.plate("N", 10, subsample_size=5, dim=-1)
+
+    local_guide = AutoNormal(model, create_plates=create_plates)
```

Comment:
is it worth checking explicitly that this gives the same result as the `local_guide=None` case?